### PR TITLE
Adding Mexico Locals

### PR DESCRIPTION
**Pull request by MapGuy11**
1. ADD Azteca 7
2. CHANGE Azteca Uno To My Multi-Stream / Multi Resolutions Server
3. ADD ImagenTV
4. ADD Imagen Multicast (Formely: Excélsior TV)
5. ADD OnceTV
6. ADD Once Niñas y Niños
7. ADD XEWT (CANAL 12)

### CODE REVIEW

**Comment by MapGuy11** [comment]
Look Good?

**Review by BellezaEmporium — COMMENTED** [review]
- Keep the original call names on tvg-id when it comes to multicast versions.
- You can still keep the original links as backup links in case you run into an issue with your host.

**Comment by MapGuy11** [comment]
@LaneSh4d0w They all go to the same place. So if the host is down it's down.

**Comment by MapGuy11** [comment]
I am going to merge.

**Comment by LaneSh4d0w** [comment]
> 
> 
> @LaneSh4d0w They all go to the same place. So if the host is down it's down.

You're telling me that your host is only redirecting to those links ?

**Comment by MapGuy11** [comment]
Yes, that is correct. It will not go through directly to the host but with a proxy before it will.

File: channels/mx.m3u [hunk 1]
```diff
@@ -5,11 +5,21 @@ https://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master.m3u8
...
+#EXTINF:-1 tvg-id="XHTITTDT.mx" tvg-name="Azteca 7" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/20551-91383.png" group-title="",Azteca 7 (XHTIT-TDT)
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtit-tdt.m3u8
+#EXTINF:-1 tvg-id="XHJKTDT.mx" tvg-name="Azteca Uno" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/1.1.png" group-title="",Azteca Uno (XHJK-TDT)
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhjk-tdt.m3u8
+#EXTINF:-1 tvg-id="XHCTTITDT.mx" tvg-name="Imagen TV" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1d/Logo_Imagen_Televisi%C3%B3n_2016.png/800px-Logo_Imagen_Televisi%C3%B3n_2016.png" group-title="",Imagen TV (XHCTTI-TDT)
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhctti-tdt.m3u8
+#EXTINF:-1 tvg-id="XHCTTITDT2.mx" tvg-name="Imagen Multicast" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwngCFclkx8_OjeImr3mO1BvDLtxIUkDcDL037PCAgw" group-title="",Imagen Multicast (XHCTTI-TDT)
```
Comment (LaneSh4d0w) [inline channels/mx.m3u:14]: Keep the original call sign.

**Comment by MapGuy11** [comment]
And that host is only an antenna in certain cities transcoding to h264 HLS URLs.

File: channels/mx.m3u [hunk 2]
```diff
@@ -5,11 +5,21 @@ https://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master.m3u8
...
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhctti-tdt2.m3u8
+#EXTINF:-1 tvg-id="XHTJBTDT.mx" tvg-name="Once TV" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/20551-48379.png" group-title="",Once TV (XHTJB-TDT)
+https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtjb-tdt.m3u8
+#EXTINF:-1 tvg-id="XHTJBTDT2.mx" tvg-name="Once Niñas y Niños" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://media.localbtv.com/images/sandiegobtv/logos/20551-80512.png" group-title="",Once Niñas y Niños (XHTJB-TDT)
```
Comment (LaneSh4d0w) [inline channels/mx.m3u:18]: Keep the original call sign.

**Comment by LaneSh4d0w** [comment]
> 
> 
> Yes, that is correct. It will not go through directly to the host but with a proxy before it will.

OK, then simply remove the 

> 
> 
> And that host is only an antenna in certain cities transcoding to h264 HLS URLs.

Noted, please see the comments I've put.

**Comment by MapGuy11** [comment]
Ok.

File: channels/mx.m3u (hunk 1 above)
Comment (MapGuy11) [reply channels/mx.m3u:14]: So keep just XHCTTITDT instead of marking the DT2?

Comment (LaneSh4d0w) [reply channels/mx.m3u:14]: Yes, that's how the channel is called.

Comment (MapGuy11) [reply channels/mx.m3u:14]: ok will change

**Review by BellezaEmporium — APPROVED** [review]

File: channels/mx.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/mx.m3u]: According to this : https://www.rabbitears.info/search.php?request=network_search&network=ImagenTV, it does.

**Comment by MapGuy11** [comment]
Ok, I am going to merge now sorry I just didn't want to merge my own work before someone else looks at it!

**Comment by LaneSh4d0w** [comment]
Better be safe than sorry !

**Comment by MapGuy11** [comment]
Yes!

**Pull request merged**